evalml/automl/automl_algorithm/iterative_algorithm.py

```diff
@@ -19,7 +19,7 @@ def __init__(self,
                  max_iterations=None,
                  tuner_class=None,
                  text_columns=None,
-                 random_state=0,
+                 random_seed=0,
```

Comment:
I'm replacing random_state with random_seed here, rather than adding it as an argument, because this is not a user-facing class.